### ISSUE TITLE
ISSUE: Bulk Actions delete all items in the table when not selected.

### ISSUE THREAD

**Issue by twoarmtom** [protonemedia/laravel-splade#266]
- Laravel Version: 9.48.0
- PHP Version: 8.1.7
- Splade JS Version (npm): 1.1.2
- Splade PHP Version (composer): 1.2.4

### Description:

When filtering a table with search criteria, and attempting to perform a Bulk Action (deleting rows), there are two options with the same number listed (which is expected behaviour):
- Select all on the this page (11)
- Select all results (11)

However, if you choose Select all results, it will perform the action on all items in the table, including unfiltered items.

### Steps To Reproduce Issue:

Filter a table using search criteria, perform the bulk action using Select all results.

Bulk Action:
```
->bulkAction(
        label: 'Delete Students',
        each: fn (User $student) => $student->deleteUser(),
        after: fn () => Toast::info('Students deleted!')->autoDismiss(3),
        confirm: 'Delete Students?',
        confirmText: 'Are you sure, this cannot be reversed?',
        confirmButton: 'Yes, delete all selected rows.',
        cancelButton: 'No',
    )
```
In my User Model
```
public function deleteUser()
    {
        return $this->delete();
    }
```

The Select all on this page works as expected, just the Select all results is not limiting to the filtered list.

**Comment by twoarmtom** [comment]
In the table component file select-rows-dropdown.blade.php

This is how the @click function is written for Select all on this page:
```
@click="table.setSelectedItems(@js($table->getPrimaryKeys()))"
```
Here's how it's written for Select all results:
```
@click="table.setSelectedItems(['*'])"
```

I'm not sure what it's suppose to be referencing, but it simply deletes everything in the table even if it's not part of the filtered data.

**Comment by pascalbaljet** [comment]
Fixed in v1.2.5, make sure to [update both packages](https://splade.dev/docs/upgrading) (JS + PHP). And thanks for your sponsorship! ❤️
